src/workerd/io/compatibility-date.capnp

```diff
@@ -109,6 +109,7 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
 
   streamsJavaScriptControllers @6 :Bool
       $compatEnableFlag("streams_enable_constructors")
+      $compatEnableDate("2022-10-31")
```

Comment:
I'd feel a bit better pushing this and `transformstream_enable_standard_constructor` back a month, until the new tee behavior has proven itself stable. What do you think?